README.md

```diff
@@ -93,3 +93,29 @@ node my-file.js
 You may need to adjust the code so the URL points towards your running QIX Engine.
 
 ![/getting-started.gif](/getting-started.gif)
+  
+You can also use a service like [unpkg](https://unpkg.com/#/) to test enigma.js directly in your browser without using node for development purposes.  
+
+Create a html file `index.html` and insert the following contents
```

Comment:
`html` -> `HTML`, this sentence suggests this exact code needs to be used, perhaps add `example` to clarify

End with `:`